### PR TITLE
bpftrace: add to debug container

### PR DESCRIPTION
**Pull request by christoph-zededa**
add `bpftrace-aotrt` to the debug container

### CODE REVIEW

**Comment by christoph-zededa** [comment]
it tried to copy an empty directory on riscv64; I hope it builds now successfully.

**Review by rouming — APPROVED** [review]

**Comment by christoph-zededa** [comment]
build failures:
```
Error: error building "lfedge/eve-debug:21943b9096888218a689ada1e1fa9a2fb4a42e79": error building for arch riscv64: failed to solve: process "/dev/.buildkit_qemu_emulator /bin/sh -c su builder -c 'cd /musl && abuild checksum && abuild -r'" did not complete successfully: exit code: 1
2024/07/15 10:54:36 error during command execution: error building "lfedge/eve-debug:21943b9096888218a689ada1e1fa9a2fb4a42e79": error building for arch riscv64: failed to solve: process "/dev/.buildkit_qemu_emulator /bin/sh -c su builder -c 'cd /musl && abuild checksum && abuild -r'" did not complete successfully: exit code: 1
make: *** [Makefile:1035: eve-debug] Error 1
```
and
```
eve-build-ubuntu docker container is ready to use
Cloning into '/tmp/linuxkit'...
error: 4504 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output
make: *** [Makefile:941: /home/ubuntu/actions-runner/_work/eve/eve/build-tools/bin/linuxkit] Error 128
```

IMO both are unrelated to this PR.

**Review by eriknordmark — CHANGES_REQUESTED** [review]
The riscv mini build still fails. Might want to rebase on master and see if that changes anything.

**Comment by christoph-zededa** [comment]
Trying something out regarding these build errors ...

**Comment by eriknordmark** [comment]
That seems to have fixed it. Now need a DCO fix.

File: pkg/debug/Dockerfile [hunk 1]
```diff
@@ -29,25 +30,28 @@ ENV LSHW_VERSION 02.19.2
...
 ENV MUSL_VERSION v1.2.3
 ENV DISTFILES_MIRROR /tmp/mirror
-ADD --chown=builder:abuild https://git.musl-libc.org/cgit/musl/snapshot/${MUSL_VERSION}.tar.gz ${DISTFILES_MIRROR}/musl-${MUSL_VERSION}.tar.gz
-RUN su builder -c 'cd /musl && abuild checksum && abuild -r'
+ADD https://git.musl-libc.org/cgit/musl/snapshot/${MUSL_VERSION}.tar.gz ${DISTFILES_MIRROR}/musl-${MUSL_VERSION}.tar.gz
+WORKDIR /musl
+RUN abuild -F checksum
+RUN abuild -F -r
 
+RUN find / -name musl-1.2.3-r0.apk
```
Comment (rene) [inline pkg/debug/Dockerfile:44]: What is the purpose of this find command? Forgot from debug?

Comment (rene) [reply pkg/debug/Dockerfile:44]: you can use an approach like here: https://github.com/lf-edge/eve/blob/master/pkg/cross-compilers/Dockerfile#L19 to fetch the whole index beforehand... but I'm wondering why we didn't experience this error before...

Comment (rene) [reply pkg/debug/Dockerfile:44]: you can also try `apk add --no-network ....`

**Review by eriknordmark — APPROVED** [review]
This appears to be building now. Any reason to keep it as a draft?

**Comment by christoph-zededa** [comment]
> This appears to be building now. Any reason to keep it as a draft?

No, I was just pessimistic ;-)

**Pull request merged**